tests/queries/test_bulk_update.py

```diff
@@ -125,15 +125,26 @@ def test_update_custom_primary_key(self):
 
     def test_empty_objects(self):
         with self.assertNumQueries(0):
-            Note.objects.bulk_update([], ['note'])
+            rows_updated = Note.objects.bulk_update([], ['note'])
+
+        self.assertEqual(rows_updated, 0)
 
     def test_large_batch(self):
         Note.objects.bulk_create([
             Note(note=str(i), misc=str(i))
             for i in range(0, 2000)
         ])
         notes = list(Note.objects.all())
-        Note.objects.bulk_update(notes, ['note'])
+        rows_updated = Note.objects.bulk_update(notes, ['note'])
+
+        self.assertEqual(rows_updated, 2000)
+
+    def test_updated_rows_when_passing_duplicate_values(self):
+        note = Note.objects.create(note='test-note', misc='test')
+
```

Comment:
```suggestion
```

Reply:
This newline is used to separate the "act" part of testing from "assert". Inspired from [AAA pattern of testing](http://wiki.c2.com/?ArrangeActAssert). Would you still want me to get rid of the new lines everywhere?

Reply:
Yes please remove unnecessary blank lines.

Reply:
i have removed them. thanks for the suggestion.